testnet-validator-example/README.md

```diff
@@ -0,0 +1,104 @@
+My Package
+============
+This is an Aptos Testnet Validator [Kurtosis package](https://docs.kurtosis.com/concepts-reference/packages). 
+It creates a local testnet with 1 validator and a Faucet to generate coins. 
+For more background on local testnets [reference Aptos' docs](https://aptos.dev/nodes/local-testnet/run-a-local-testnet). 
+
+Run this package
+----------------
+If you have [Kurtosis installed][install-kurtosis], run:
+
+<!-- TODO replace YOURUSER and THISREPO with the correct values -->
+```bash
+kurtosis run github.com/kurtosis-tech/aptos-package/testnet-validator-example --enclave aptos
+```
+
+If you don't have Kurtosis installed, [click here to run this package on the Kurtosis playground](https://gitpod.io/#KURTOSIS_PACKAGE_LOCATOR=/https://github.com/kurtosis-tech/playground-gitpod).
```

Comment:
replace `/https://github.com/kurtosis-tech/playground-gitpod` with this package's URL